src/services/uploader/uploading.controller.ts

```diff
@@ -152,7 +152,7 @@ export class UploadingController {
   }
 
   @UseGuards(AuthGuard('jwt'))
-  @UseGuards(AuthGuard('jwt'), RequireHiveVerify)
+  @UseGuards(AuthGuard('jwt'))
```

Comment:
```suggestion
```

Reply:
this will need authentication, it just looks like its not implemented yet

Reply:
Same as https://github.com/spknetwork/acela-core/pull/28#discussion_r1658908308